django/db/models/base.py

```diff
@@ -324,7 +326,7 @@ def _prepare(cls):
 
             # defer creating accessors on the foreign class until we are
             # certain it has been created
-            def make_foreign_order_accessors(field, model, cls):
+            def make_foreign_order_accessors(cls, model, field):
```

Comment:
Oh, never mind, now I see - this is a callback passed to `lazy_related_operation`. Yeah, changing the order of arguments of an internal callback is not an issue - it's not even possible for anyone external to call this function.